pkg/component/input_spec.go

```diff
@@ -12,13 +12,14 @@ import (
 
 // InputSpec is the specification for an input type.
 type InputSpec struct {
-	Name        string      `config:"name" yaml:"name"  validate:"required"`
-	Aliases     []string    `config:"aliases,omitempty" yaml:"aliases,omitempty"`
-	Description string      `config:"description" yaml:"description" validate:"required"`
-	Platforms   []string    `config:"platforms" yaml:"platforms" validate:"required,min=1"`
-	Outputs     []string    `config:"outputs,omitempty" yaml:"outputs,omitempty"`
-	Shippers    []string    `config:"shippers,omitempty" yaml:"shippers,omitempty"`
-	Runtime     RuntimeSpec `config:"runtime,omitempty" yaml:"runtime,omitempty"`
+	Name           string      `config:"name" yaml:"name"  validate:"required"`
+	Aliases        []string    `config:"aliases,omitempty" yaml:"aliases,omitempty"`
+	Description    string      `config:"description" yaml:"description" validate:"required"`
+	Platforms      []string    `config:"platforms" yaml:"platforms" validate:"required,min=1"`
+	Outputs        []string    `config:"outputs,omitempty" yaml:"outputs,omitempty"`
+	ProxiedActions []string    `config:"proxied_actions,omitempty" yaml:"proxied_actions,omitempty"`
```

Comment:
This adds a new field to the input spec format -- please document this field in `docs/component-specs.md`, it's not obvious what its intended effect is

Reply:
Done